drivers/sensor/sensor_shell.c

```diff
@@ -46,20 +46,32 @@ const char *sensor_channel_name[SENSOR_CHAN_ALL] = {
 	[SENSOR_CHAN_DISTANCE] =	"distance",
 	[SENSOR_CHAN_CO2] =		"co2",
 	[SENSOR_CHAN_VOC] =		"voc",
+	[SENSOR_CHAN_GAS_RES] =         "gas_resistance",
```

Comment:
nit: please replace the spaces with tabs to be consistent with the rest of the file